roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -39,52 +38,61 @@ metadata:
     {{ lookup("template", "../common/templates/labels/common.yaml.j2") | indent(width=4) | trim }}
 data:
   nginx_default_conf_template: |
-    server {
-        listen 8080;
-        {% if not ipv6_disabled %}
-        listen [::]:8080;
-        {% endif %}
-
-        server_name _;
-        server_tokens off;
-
-        access_log off;
-        # error_log off;
-
-        autoindex off;
-
+    events {
+        worker_connections 1024;
+    }
+    http {
         include mime.types;
         types {
             application/manifest+json webmanifest;
         }
+        server {
+            listen 8080;
+            {% if not ipv6_disabled %}
+            listen [::]:8080;
+            {% endif %}
 
-        sendfile on;
+            server_name _;
+            server_tokens off;
 
-        root /usr/share/nginx/html;
+            access_log off;
+            # error_log off;
 
-        location ~ ^/api/eda/v[0-9]+/ {
-            proxy_pass $EDA_SERVER;
-            proxy_set_header Origin $EDA_SERVER;
-        }
+            autoindex off;
 
-        location ~ ^/api/eda/ws/[0-9a-z-]+ {
-            proxy_pass $EDA_SERVER;
-            proxy_set_header Origin $EDA_SERVER;
-            proxy_http_version 1.1;
-            proxy_set_header Upgrade $http_upgrade;
-            proxy_set_header Connection "Upgrade";
-        }
+            include mime.types;
+            types {
+                application/manifest+json webmanifest;
+            }
 
-        location ~* \.(json|woff|woff2|jpe?g|png|gif|ico|svg|css|js)$ {
-            add_header Cache-Control "public, max-age=31536000, s-maxage=31536000, immutable";
-            try_files $uri =404;
-            gzip_static on;
-        }
+            sendfile on;
 
-        location / {
-            autoindex off;
-            expires off;
-            add_header Cache-Control "public, max-age=0, s-maxage=0, must-revalidate" always;
-            try_files $uri /index.html =404;
-        }
+            root /usr/share/nginx/html;
+
+            location ~ ^/api/eda/v[0-9]+/ {
+                proxy_pass http://{{ ansible_operator_meta.name }}-api:8000;
```

Comment:
Why are we not passing the rest of the headers as we do in RHEL? 
```
        proxy_set_header Host $http_host;
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header X-Forwarded-Port $server_port;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;

Reply:
@Alex-Izquierdo I opened another PR here for that just now:
* https://github.com/ansible/eda-server-operator/pull/143

Can you take a look there and make sure that it covers everything?